app/model.rb

```diff
@@ -17,15 +18,19 @@ class User
 DataMapper.finalize.auto_upgrade!
 
 class Ocr
-  def init
-    @engine = Tesseract::Engine.new {|engine|
-                engine.language = :lol # arbitrary
+  def initialize
+    @engine = ::Tesseract::Engine.new {|engine|
+                engine.language = :eng
                 engine.page_segmentation_mode = 4
-                engine.whitelist = [*'a'..'z', *'A'..'Z', *0..9].join # thanks ruby, made this easy
+                engine.whitelist = [*'a'..'z', *'A'..'Z', *0..9].flatten # thanks ruby, made this easy
```

Comment:
And yes I used meh's code as a base just to toy in the committed pieces but have been using a combination of the executable provided with the gem and a default settings instantiation not wrapped in the class to experiment. As I said, super rough so I'm just trying random angles atm.